Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs

```diff
@@ -250,8 +249,8 @@ public DomainProfile(string email, string firstName, string lastName)
             FirstName = firstName;
             LastName = lastName;
             ProfilePictureUrl = new Uri(ProfileConfiguration.DefaultUri);
-            FollowerEmails = new List<string>();
-            FollowingEmails = new List<string>();
+            Followers = new List<DomainProfile>();
+            Following = new List<DomainProfile>();
```

Comment:
if you want to store the follower object reference like this, we need to be able to distinguish between "the profile has zero followers" and "the profile might have followers but we haven't loaded them". otherwise, you'd be forced to load the entire network of profiles from the DB to look at just one of them.